src/oc_erchef/rebar.config

```diff
@@ -60,17 +59,19 @@
         {darklaunch, ".*",
          {git, "https://github.com/chef/opscode-darklaunch-erlang", {branch, "master"}}},
         {opscoderl_httpc, ".*",
-         {git, "https://github.com/chef/opscoderl_httpc", {branch, "master"}}},
+         {git, "https://github.com/chef/opscoderl_httpc", {branch, "ma/ibrowse_uses_mainline"}}},
         {sync, ".*",
          {git, "https://github.com/rustyio/sync.git", {branch, "master"}}},
         {prometheus, ".*",
          {git, "https://github.com/deadtrickster/prometheus.erl", {tag, "v3.4.0"}}},
+        {eper, ".*",
+         {git, "https://github.com/massemanet/eper", {branch, "master"}}},
         {observer_cli, ".*",
          {git, "https://github.com/zhongwencool/observer_cli", {branch, "master"}}}
        ]}.
 
-{cover_enabled, true}.
-{cover_print_enabled, true}.
+%{cover_enabled, true}.
+%{cover_print_enabled, true}.
```

Comment:
can delete the commented section if it is not wanted.

Reply:
commenting those out was a mistake.  good catch!